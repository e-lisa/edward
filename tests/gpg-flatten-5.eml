From: No One <noone@localhost>
MIME-Version: 1.0
To: No One <noone@localhost>
Subject: =?UTF-8?B?44GT44KT44Gr44Gh44Gv44CB44GK5YWD5rCX44Gn44GZ44GL77yf?=
Content-Type: multipart/signed; micalg=pgp-sha512;
 protocol="application/pgp-signature";
 boundary="69pEqCNMs7DnfdpwkAdpUCaf0lwnKu6Vr"

This is an OpenPGP/MIME signed message (RFC 4880 and 3156)
--69pEqCNMs7DnfdpwkAdpUCaf0lwnKu6Vr
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

thanks for the message!

> =C3=9Cber Spa=C3=9F.
>
> qual =C3=A8 il suono di una mano sola?
>
> =E3=81=93=E3=82=93=E3=81=AB=E3=81=A1=E3=81=AF=E3=80=81=E3=81=8A=E5=85=83=
=E6=B0=97=E3=81=A7=E3=81=99=E3=81=8B=EF=BC=9F
>

test.

-andrew









--69pEqCNMs7DnfdpwkAdpUCaf0lwnKu6Vr
Content-Type: application/pgp-signature; name="signature.asc"
Content-Description: OpenPGP digital signature
Content-Disposition: attachment; filename="signature.asc"

-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1

iJwEAAECAAYFAlZcv6cACgkQsT5tL2pPbuiVHAP/fl0fViiYtc5UHAOzywlTRKP0
DXCp4mgwkeqawaC4+N7CN1I4dezoWBoKcOBJRuF6rrkeet7vsj8FeJAvzBTui6PI
XxGPU4wuaaQ0R9DlePqKPzScrDEBXvxvJdk9AUilkiU3/jaCsgpalAaLrwK4PCIL
iXisvPWXi2u7tSPpJWU=
=GWbS
-----END PGP SIGNATURE-----

--69pEqCNMs7DnfdpwkAdpUCaf0lwnKu6Vr--

