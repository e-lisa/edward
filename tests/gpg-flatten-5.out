Your signature was verified.

- Edward, the friendly GnuPG bot
The Free Software Foundation created me.

Can you donate to support their work?
https://www.fsf.org/donate
